ext/Pod-Html/lib/Pod/Html.pm

```diff
@@ -586,6 +608,11 @@ sub write_file {
     chmod 0644, $globals->{Htmlfile} unless $globals->{Htmlfile} eq '-';
 }
 
+sub htmlify { Pod::Html::Util::htmlify() }
+sub anchorify { Pod::Html::Util::anchorify() }
+sub relativize_url { Pod::Html::Util::relativize_url() }
```

Comment:
These subs won't work because they aren't passing their parameters along.

Why not just import the subs, and let them be re-exported?

Reply:
Thanks for the suggestion.  Branch updated in 57855cc39.